lib/realtime/api/presence.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Realtime.Api.Presence do
+  @moduledoc """
+  Defines the Presence schema
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+  @derive {Jason.Encoder, only: [:inserted_at, :updated_at, :id, :channel_id]}
+
+  @type t :: %__MODULE__{}
+
+  @schema_prefix "realtime"
+  schema "presences" do
+    field(:check, :boolean, default: false)
+    timestamps()
+
+    belongs_to(:channel, Realtime.Api.Channel)
+  end
+
+  def changeset(broadcast, attrs) do
+    broadcast
+    |> cast(attrs, [:check, :inserted_at, :updated_at, :channel_id])
+    |> put_timestamp(:updated_at)
+    |> maybe_put_timestamp(:inserted_at)
```

Comment:
I'm sure it has came up before but double-checking, do we need to manually handle the timestamps? Ecto should be doing that already.

Reply:
Since we're not using Ecto to directly connect to the database and only Postgrex we end up needed to handle this unfortunately